<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>film_ekle_button</class>
 <widget class="QDialog" name="film_ekle_button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>492</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-size:20px;
font-style:italic;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Film Ekleme Paneli</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>431</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="ana_menu_buton">
      <property name="styleSheet">
       <string notr="true">background-color:white;font-size:15px;color:red;font-weight:bold;font-style:italic;</string>
      </property>
      <property name="text">
       <string>Ana Menü</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/newPrefix/images/menu.png</normaloff>:/newPrefix/images/menu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cikis_buton">
      <property name="styleSheet">
       <string notr="true">background-color:white;font-size:15px;color:red;font-weight:bold;font-style:italic;</string>
      </property>
      <property name="text">
       <string>Çıkış</string>
      </property>
      <property name="icon">
       <iconset resource="img.qrc">
        <normaloff>:/newPrefix/images/exit.png</normaloff>:/newPrefix/images/exit.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="film_ekle_buton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>431</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; 
color:red;
font-size:15px;font-style:italic;font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Film Ekle</string>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/newPrefix/images/save.png</normaloff>:/newPrefix/images/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="film_afisi_label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>181</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
padding-left:40px;
font-size:16px;
font-weight:bold;
font-style:italic;</string>
   </property>
   <property name="text">
    <string>Film Afişi</string>
   </property>
  </widget>
  <widget class="QLabel" name="seans_bilgi">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;font-size:15px;padding-left:150px;</string>
   </property>
   <property name="text">
    <string>___Seans Bilgisi___</string>
   </property>
  </widget>
  <widget class="QLabel" name="film_adi_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="text">
    <string>Film Adı:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="film_ismi_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="film_turu_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="text">
    <string>Film Türü:</string>
   </property>
  </widget>
  <widget class="QLabel" name="yonetmen">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="text">
    <string>Yönetmen:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yonetmen_text">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="salon_label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>268</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="text">
    <string>Salon:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="salon_combo_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>267</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="afis_label">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>222</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Afiş:</string>
   </property>
  </widget>
  <widget class="QLabel" name="seans_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>300</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white; font-size:13px;</string>
   </property>
   <property name="text">
    <string>Seans:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="seans_combo_box">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="afis_ekle_buton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
   <property name="text">
    <string>Afiş Ekle</string>
   </property>
  </widget>
  <widget class="QComboBox" name="film_turu_combo_box">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
